src/duke/Deadline.java

```diff
@@ -0,0 +1,13 @@
+package duke;
+
+public class Deadline extends Task {
+
+    Deadline(String name, String deadline) {
+        super(name, deadline);
+    }
+
+    @Override
+    String getIndicator() {
```

Comment:
Do you expect this method to have a specified visibility, e.g. `public`? I will omit similar problems of missing specification of visibility for other files.